main.go

```diff
@@ -60,6 +60,14 @@ type (
 	}
 
 	VaultAllShares map[string][]*keygen.LocalPartySaveData
+
+	AppConfig struct {
+		filenames      []string
+		nonceOverride  int
+		quorumOverride int
+		exportKSFile   string
+		passwordForKS  string
+	}
```

Comment:
@notatestuser we could pass this structure to `runTool`, WDYT?

Reply:
Not necessary imo